packages/renderer/src/lib/container/ContainerList.svelte

```diff
@@ -467,16 +467,22 @@ function setStoppedFilter() {
   </svelte:fragment>
   <svelte:fragment slot="bottom-additional-actions">
     {#if selectedItemsNumber > 0}
-      <Button
-        on:click="{() => deleteSelectedContainers()}"
-        aria-label="Delete selected containers and pods"
-        title="Delete {selectedItemsNumber} selected items"
-        bind:inProgress="{bulkDeleteInProgress}"
-        icon="{faTrash}" />
-      <Button
-        on:click="{() => createPodFromContainers()}"
-        title="Create Pod with {selectedItemsNumber} selected items"
-        icon="{SolidPodIcon}" />
+      <div style="display: inline-flex;">
+        <Button
+          on:click="{() => deleteSelectedContainers()}"
+          aria-label="Delete selected containers and pods"
+          title="Delete {selectedItemsNumber} selected items"
+          bind:inProgress="{bulkDeleteInProgress}"
+          icon="{faTrash}">
+        </Button>
+        &nbsp;
```

Comment:
if you want to add space between element, maybe `space-x-2` on the parent div

https://tailwindcss.com/docs/space

Reply:
I've added the class (nbsp did feel funky) and pushed.